java/src/jmri/jmrit/dispatcher/AutoActiveTrain.java

```diff
@@ -221,6 +221,14 @@ public float getMaxTrainLength() {
         return _maxTrainLength;
     }
 
+    public int getMaxTrainLengthMM() {
+        double dMM = _maxTrainLength * dispatcher.getScale().getScaleFactor();
+        if (dispatcher.getUseScaleMeters()) {
+            return (int)(dMM  * 1000.0);
+        }
+        return (int)(dMM / 0.00328084);
```

Comment:
No need to change, but it helps other developers if constants like this one is documented. A recommendation is to have it as a constant:

public static final double MY_CONSTANT = 0.123456;